Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp

```diff
@@ -36,7 +37,8 @@
 
 struct _WebKitOpenCDMDecryptPrivate {
     String m_session;
-    std::unique_ptr<media::OpenCdm> m_openCdm;
+    struct OpenCDMAccessor* m_openCdmAccessor;
+    struct OpenCDMSession* m_openCdm;
```

Comment:
Do you need the `struct` here?

Another question is why can't you use here a `unique_ptr` and avoid the manual destruction later?